docs/dev_guide/diagnostic_ref/dpct1038.rst

```diff
@@ -23,4 +23,65 @@ parameters. This may lead to incorrectly generated |dpcpp| code.
 Suggestions to Fix
 ------------------
 
-Review the kernel call inside the macro definition and adjust it manually, if needed.
\ No newline at end of file
+Review the kernel call inside the macro definition and adjust it manually, if needed.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __global__ void kernel(int c, int d) {}
+    
+    void foo() {
+    #define KERNEL_LAUNCH(NAME, a, b, c, d) NAME<<<a, b, 0>>>(c, d);
+      KERNEL_LAUNCH(kernel, 3, 2, 1, 0)
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done